package.json

```diff
@@ -19,6 +19,7 @@
   },
   "devDependencies": {
     "fantasy-land": "3.5.0",
+    "list": "2.0.7",
```

Comment:
@Avaq, should this be listed here or in __bench/package.json__?

Reply:
It's used as a library like any other (just so happens to be used in benchmarks), so it should be listed here. The `bench/package.json` file only exists to allow for the installation of any version of choice, of the package you're benchmarking against.